language/sygus.tex

```diff
@@ -1524,6 +1549,41 @@ \section{Synthesis Solver Output}%
 was unable to find a solution, which does not necessarily imply that
 the conjecture is infeasible.
 
+\item
+A (successful) well-formed response to $\paren{\optsynthkwd\mbox{ }\paren{t_1 \ldots t_j}\mbox{ } A_1 \ldots A_k}$
+is a tuple of terms followed list of commands, enclosed in parentheses, of the form:
+\[
+\begin{array}{l}
+\paren{\\
+\paren{c_1 \ldots c_j}\\
+\paren{\fundefcmdkwd\mbox{ }f_{1}\mbox{ }X_{1}\mbox{ }\sigma_{1}\mbox{ }t_{1}}\\
+\ldots\\
+\paren{\fundefcmdkwd\mbox{ }f_{n}\mbox{ }X_{n}\mbox{ }\sigma_{n}\mbox{ }t_{n}}\\
+}
```

Comment:
Note that $t_1$ is overloaded here.

May be use $t'_1, \ldots, t'_j$ within `optimize-synth`?